packages/react-scripts/fixtures/kitchensink/src/App.js

```diff
@@ -116,8 +121,9 @@ class App extends React.Component {
   }
 
   render() {
-    const Feature = this.state.feature;
-    return Feature ? <BuiltEmitter><Feature /></BuiltEmitter> : null;
+    const { feature } = this.state
+    if (feature !== null) return <BuiltEmitter feature={feature} />
```

Comment:
A few style nits:

* Use semis
* Please always use multiline `if`
* Keep JSX balanced for multiline JSX, i.e. put `(`, then JSX opening tag on next line